controllers/goharbor/notaryserver/deployments.go

```diff
@@ -107,24 +103,41 @@ func (r *Reconciler) GetDeployment(ctx context.Context, notary *goharborv1alpha2
 	}
 
 	initContainers := []corev1.Container{}
+	migrateCmd := ""
+	migrationEnvs := []corev1.EnvVar{}
+
+	if notary.Spec.MigrationEnabled == nil || *notary.Spec.MigrationEnabled {
```

Comment:
default is true?

Reply:
not set means migration is enabled?

Reply:
yes, the default is true, or fresh installation will be fail